src/tags/if_block.rs

```diff
@@ -185,7 +185,40 @@ impl Renderable for Conditional {
     }
 }
 
-fn parse_atom_condition(arguments: &mut TagTokenIter) -> Result<Condition> {
+struct PeekableTagTokenIter<'a> {
```

Comment:
I don't really think this is the best approach. However, I wasn't able to find another way to have both `peek` and `expect_next` accessible to parse the condition.